.github/workflows/tests.yml

```diff
@@ -100,9 +101,9 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: [7.4, 8.0, 8.1, 8.2, 8.3]
```

Comment:
Are we intentionally not testing 8.1-8.3?

Reply:
Yea we're figuring 7.4, 8.0, and 8.4 will be enough coverage and the reduced time to complete a full suite would be a benefit.  We have the ability on the minimal-tests now to run them on demand and specify a version though if we have reason to believe a specific 8.x minor version is having a problem